apps/core/src/utils/getRefGasPrice.ts

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { SuiValidatorSummary } from '@mysten/sui.js';
+
+const REF_THRESHOLD = 66.67;
+
+/**
+ * Util to get the Reference Gas Price from a list of validators
+ * 1. Sort validators by gas price
+ * 2. Calculate the stake threshold at 66.67% of total stake amount
+ * 3. Add up the stake amount of each validator until the threshold is reached
+ * 4. Return the gas price of the last validator that was added to the sum
+ */
+export function getRefGasPrice(validators?: SuiValidatorSummary[]) {
```

Comment:
seems weird to me that this returns a string, I feel like a bigint is a better return value